.github/actions/analyse-obs/action.yaml

```diff
@@ -0,0 +1,60 @@
+name: Run PVS-Studio analysis
+inputs:
+  pvsUsername:
+    description: PVS Studio Licence Username
+    required: true
+  pvsKey:
+    description: PVS Studio Licence Key
+    required: true
+  target:
+    description: Build Target
+    required: true
+  config:
+    description: Build Configuration
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: 'Setup PVS Studio'
+      shell: pwsh
+      run: |
+        Invoke-WebRequest -Uri https://cdn.pvs-studio.com/PVS-Studio_setup.exe -OutFile ..\PVS-Studio_setup.exe
```

Comment:
We should use hash checks here as with all other external code and binaries we download in our CI workflows/actions.

Reply:
Unfortunately they don't seem to have versioned download URLs, so this would break every time they change it. So a hash could be set as non-secret actions variable, or we could rely on the code signature (via `Get-AuthenticodeSignature`).